std_msgs/Bool visible
std_msgs/String color